src/facets/BoostFacet.sol

```diff
@@ -39,11 +40,18 @@ contract BoostFacet {
         if (isStratosphereMember) {
             _boostFee = s.boostLevelToFee[boostLevel];
         }
+        _userData.lastBoostClaimTimestamp = block.timestamp;
+        uint256 _boostPointsAmount;
```

Comment:
this variable is not storing points amount but the boost percentage, how about we rename it `boostPercent`?